index.js

```diff
@@ -28,16 +28,16 @@ function message (commitText) {
   if (isNewline(scanner.peek())) {
     scanner.next()
   } else {
-    invalidToken(scanner, ['none'])
+    return invalidToken(scanner, ['none'])
```

Comment:
😄

Reply:
whoops, screwed up the refactor.

Reply:
haha no problem, I did so too with this return. It should be a `throw` because this is from the `message` tokenizer (root)